docs/cookbook/migrations/homebrew.md

```diff
@@ -0,0 +1,178 @@
+* `init` creates a new environment
+* `activate` activates an environment
+
+### The default environment
+Typically, a Flox user will create a [default environment]() in their home directory that is activated each time a new shell is created. This is useful for packages that should be available to the user across all of the contexts where they work.
+
+When Flox is configured with a default environment, it behaves very similarly to Homebrew. The Flox CLI will assume the default environment when you call `flox install` commands.
+
+Creating a Flox default environment is optional.
+
+## Install Flox
+Download and install Flox following the [installation instructions]().
+
+## Find and install your first package
+This section walks you through the process for finding your installed Homebrew packages in the Flox Catalog.
+
+### Show top-level formulae in Homebrew
+First, identify the list of Homebrew packages you have installed.
+
+We recommend using `brew leaves` for this, so you can easily differentiate between the formulae you installed explicitly versus those that were installed as dependencies. The `leaves` subcommand will show the formulae that were directly installed.
+
+For example:
+
+```
+% brew leaves
+boost
+gh
+imagemagick
+mysql
+openssh
+```
+
+Then, search Flox for one of the formulae you have installed with Homebrew. In this case, for example, you could choose `boost`:
+
+```
+% flox search boost
+boost     Collection of C++ libraries
+booster   Fast and secure initramfs generator
+xgboost   Scalable, Portable and Distributed Gradient Boosting (GBDT, GBRT or GBM) Library
+boost155  Collection of C++ libraries
+boost159  Collection of C++ libraries
+boost15x  Collection of C++ libraries
+boost160  Collection of C++ libraries
+boost165  Collection of C++ libraries
+boost166  Collection of C++ libraries
+boost167  Collection of C++ libraries
+
+Showing 10 of 120 results. Use `flox search boost --all` to see the full list.
+
+Use 'flox show <package>' to see available versions
+```
+
+The first one on the list is the correct Flox package to install, and it has the same name as the Homebrew package. This will often be the case, but not always.
+
+### Install your first package
+To install your first package, use `flox install`:
+
+```
+% flox install boost
+```
+
+The first time you install a package, Flox will ask you whether you want to create a default environment. If you choose to do so, Flox will then ask you whether you want to configure your dotfiles to automatically activate the new default environment.
+
+```
+[show the flow]
+```
+
+If you do not choose for this to be automated at the time of your first package installation, you can [follow these instructions]() to add Flox to your dotfiles manually.
+
+### Verifying configuration
+Exit your active shell and create a new one, causing the dotfile changes to take effect. The first time this happens, you may experience a short delay while your default environment is materialized.
```

Comment:
```suggestion
Exit your active shell and create a new one, causing the dotfile changes to take effect. The first time this happens, you may experience a short delay while your default environment is materialized. The next time you open a shell it should be quick because the environment has been cached.
```